indigo/distribution.yaml

```diff
@@ -1953,7 +1953,12 @@ repositories:
       tags:
         release: release/indigo/{package}/{version}
       url: https://github.com/yujinrobot-release/flask-ext-catkin-release.git
-      version: 0.1.0-0
+      version: 0.1.0-1
+    source:
+      type: git
+      url: https://github.com/asmodehn/flask-ext-catkin.git
+      version: 0.1.0
```

Comment:
Yes it must be a branch for now since Jenkins can't handle a tag. `master` should work for your repo.